### PR TITLE
Feature#6/result UI

### LINKED ISSUE #6: Results Screen UI Design
**REQUIREMENTS**
- Implement UI design in this screen
- Connect with other screens

**DESIGN**
Captura de Pantalla 2021-03-23 a la(s) 12 41 37

**ACCEPTANCE CRITERIA**
- [ ] Same design as provided.
- [ ] Good code quality.

### PR DESCRIPTION
**Pull request by GracielaJazmin**
# Description
Implement of design in the screen of result, also implement of functionality in this screen
 
  ## Screenshots 
Captura de Pantalla 2021-03-29 a la(s) 12 42 16

### CODE REVIEW

File: AlertaCovid/src/components/atoms/ResultAtoms/DateDone.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React, {useState} from 'react';
+import {View, Text, TouchableOpacity} from 'react-native';
+import {constants} from '../../../utils';
+import {ResultStyles} from '../../../styles';
+import DateTimePicker from '@react-native-community/datetimepicker';
+import moment from 'moment'
+
+const DateDone = () => {
+  const [date, setDate] = useState(new Date(1598051730000));
```
Comment (LeyaimMagma) [inline AlertaCovid/src/components/atoms/ResultAtoms/DateDone.js]: what this data is hardcoded?

File: AlertaCovid/src/components/atoms/ResultAtoms/DateDone.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      </TouchableOpacity>
+      {show && (
+        <DateTimePicker
+          testID="dateTimePicker"
+          value={new Date()}
+          mode={'date'}
+          is24Hour={true}
+          display="default"
+          onChange={(e) => {
+            console.log(e.nativeEvent.timestamp);
```
Comment (LeyaimMagma) [inline AlertaCovid/src/components/atoms/ResultAtoms/DateDone.js]: remove this console.log()

File: AlertaCovid/src/utils/constants/index.js [hunk 3]
```diff
@@ -17,7 +17,15 @@ export const constants = {
...
-    submit: 'Submit'
+    submit: 'Submit',
+    DateOfTest: 'Date the test was done',
+    DateOfResult: 'Result date',
+    TypeOfTest: 'Type of test',
+    TestResult: 'Test result',
+    SymptomsDayTest: 'Did you have symptoms on the day of the test?',
+    result: 'Test result'
+
+
```
Comment (LeyaimMagma) [inline AlertaCovid/src/utils/constants/index.js]: Remove this blank space

**Review by LeyaimMagma — CHANGES_REQUESTED** [review]

File: AlertaCovid/src/components/atoms/ResultAtoms/DateResult.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        onPress={() => setShow(true)}
+        style={ResultStyles.Calender}>
+        <Text style={{padding: 10}}>{moment(date).format('DD/MM/YYYY')}</Text>
+      </TouchableOpacity>
+      {show && (
+        <DateTimePicker
+          testID="dateTimePicker"
+          value={new Date()}
+          mode={'date'}
+          is24Hour={true}
```
Comment (fernandoamz) [inline AlertaCovid/src/components/atoms/ResultAtoms/DateResult.js:24]: Review the best practices to name variables.

**Review by fernandoamz — DISMISSED** [review]

File: AlertaCovid/src/components/atoms/ResultAtoms/DateDone.js (hunk 1 above)
Comment (GracielaJazmin) [reply AlertaCovid/src/components/atoms/ResultAtoms/DateDone.js]: It was an example, and we forgot to modify the data

**Review by LeyaimMagma — APPROVED** [review]

**Review by ChantalRamirez — APPROVED** [review]
Excelent job

**Review by mariam-rivera — APPROVED** [review]
Nice job guys! 👍

**Pull request merged**